include/xrpl/protocol/detail/ledger_entries.macro

```diff
@@ -22,7 +22,8 @@
 #endif
 
 #ifndef LEDGER_ENTRY_DUPLICATE
-#define LEDGER_ENTRY_DUPLICATE(...) LEDGER_ENTRY(__VA_ARGS__)
+#define EXPAND(x) x
```

Comment:
Consider adding a comment here linking the Stack Overflow answer.

Reply:
Added in [d67b7bf](https://github.com/XRPLF/rippled/pull/5202/commits/d67b7bf138937061075f27373987d633e937c89c)